Validation/HGCalValidation/python/PostProcessorHGCAL_cfi.py

```diff
@@ -24,22 +24,18 @@
 
 eff_multiclusters = ["effic_eta 'MultiCluster Efficiency vs #eta' Num_CaloParticle_Eta Denom_CaloParticle_Eta"]
 eff_multiclusters.extend(["effic_phi 'MultiCluster Efficiency vs #phi' Num_CaloParticle_Phi Denom_CaloParticle_Phi"])
-eff_multiclusters.extend(["duplicate_eta 'MultiCluster Duplicate(Split) Rate vs #eta' NumDup_MultiCluster_Eta Denom_MultiCluster_Eta"])
-eff_multiclusters.extend(["duplicate_phi 'MultiCluster Duplicate(Split) Rate vs #phi' NumDup_MultiCluster_Phi Denom_MultiCluster_Phi"])
+eff_multiclusters.extend(["duplicate_eta 'MultiCluster Duplicate(Split) Rate vs #eta ' NumDup_MultiCluster_Eta Denom_MultiCluster_Eta"])
+eff_multiclusters.extend(["duplicate_phi 'MultiCluster Duplicate(Split) Rate vs #phi ' NumDup_MultiCluster_Phi Denom_MultiCluster_Phi"])
 eff_multiclusters.extend(["fake_eta 'MultiCluster Fake Rate vs #eta' Num_MultiCluster_Eta Denom_MultiCluster_Eta fake"])
-eff_multiclusters.extend(["fake_phi 'MultiCluster Fake Rate vs #phi' Num_MultiCluster_Phi Denom_MultiCluster_Phi fake"])
+eff_multiclusters.extend(["fake_phi 'MultiCluster Fake Rate vs #phi  Num_MultiCluster_Phi Denom_MultiCluster_Phi fake"])
```

Comment:
Not sure why the ' is removed here in this case...

Reply:
@jfernan2 thanks for spotting: that's most likely  unless the `DQMGenericClient` has a really good and smart parser...

Reply:
@jfernan2 @rovere The honest truth is that I had also seen that after checking that the relevant plot is not produced but then a couple of emails arrived and I diverted. I am getting old. Let me correct this and push a commit.